docs/DeveloperGuide.md

```diff
@@ -1215,7 +1254,15 @@ testers are expected to do more *exploratory* testing.
 
 ### Clearing all projects
 
-1. _{ more test cases …​ }_
+1. Clearing all projects in the list.
```

Comment:
Perhaps consider using "are cleared" instead of "will be cleared"; applies to clear as well